llarp/config/config.cpp

```diff
@@ -1128,20 +1133,22 @@ namespace llarp
     conf.defineOption<std::string>(
         "api",
         "bind",
-        DefaultRPCBindAddr,
-        [this](std::string arg) {
-          if (arg.empty())
+        DefaultRPCBind,
+        MultiValue,
+        [=, first = true](std::string arg) mutable {
```

Comment:
dont capture with `=` here, use `this` as the former will copy construct things we dont want to like the parent function's arguments.

Reply:
It won't copy things you don't use, but it's also not the best practice just because you aren't being explicit about the things you want to use and might unintentionally copy things outside the lambda you didn't mean to.

Reply:
I'll use `=` (or `&`) sometimes, but generally only if it's a tiny lambda of a one or two where the copied/references are obvious.

Reply:
fixed